builder/image/templates/builder

```diff
@@ -75,50 +80,14 @@ if [ -f Dockerfile ]; then
     USING_DOCKERFILE=true
 fi
 
-# pull config from controller to be used during build
-URL="{{ .deis_controller_protocol }}://{{ .deis_controller_host }}:{{ .deis_controller_port }}/v1/hooks/config"
-RESPONSE=$(/app/bin/get-app-config -url="$URL" -key="{{ .deis_controller_builderKey }}" -user=$USER -app=$APP_NAME)
-CODE=$?
-
-if [ $CODE -ne 0 ]; then
-    puts-warn $RESPONSE
-    exit 1
-fi
-
-BUILD_OPTS=()
-BUILD_OPTS+='/usr/bin/docker'
-BUILD_OPTS+=' run '
-# get application configuration
-BUILD_OPTS+=$(echo $RESPONSE | /app/bin/get-app-values)
```

Comment:
Shouldn't the `ENV_DIR` be used for that? https://devcenter.heroku.com/articles/buildpack-api#bin-compile-summary